.github/workflows/build-and-run-node-test.yaml

```
@@ -0,0 +1,40 @@
+---
+name: "Build and run node-test"
```

```suggestion
name: "Build and test"
```

Since we also test the TSS here, not just the node